### PR TITLE
[ISSUE #6121] Optimize some code style in store module

### PR DESCRIPTION
**Pull request by TheR1sing3un**
close: https://github.com/apache/rocketmq/issues/6121

**Make sure set the target branch to `develop`**

## What is the purpose of the change

fix #<xxx> 

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

**Review by RongtongJin — APPROVED** [review]

File: store/src/main/java/org/apache/rocketmq/store/ha/DefaultHAClient.java [hunk 1]
```diff
@@ -35,11 +35,26 @@
...
+     * ┌───────────────────────┬───────────────────────┐
+     * │                  slaveMaxOffset               │
+     * │                    (8bytes)                   │
+     * ├───────────────────────┴───────────────────────┤
+     * │                                               │
+     * │                  Report Header                │
+     * </pre>
+     * <p>
+     */
+    public static final int REPORT_HEADER = 8;
```
Comment (Oliverwqcwrw) [inline store/src/main/java/org/apache/rocketmq/store/ha/DefaultHAClient.java:51]: How about to use REPORT_HEADER_SIZE?

Comment (TheR1sing3un) [reply store/src/main/java/org/apache/rocketmq/store/ha/DefaultHAClient.java:51]: seems better!

Comment (TheR1sing3un) [reply store/src/main/java/org/apache/rocketmq/store/ha/DefaultHAClient.java:51]: > How about to use REPORT_HEADER_SIZE?

seems better!

File: store/src/main/java/org/apache/rocketmq/store/ha/DefaultHAConnection.java [hunk 2]
```diff
@@ -31,6 +31,22 @@
...
+     * <pre>
+     * ┌───────────────────────┬───────────────────────┬───────────────────────┐
+     * │                  physicOffset                 │         bodySize      │
+     * │                    (8bytes)                   │         (4bytes)      │
+     * ├───────────────────────┴───────────────────────┴───────────────────────┤
+     * │                                                                       │
+     * │                           Transfer Header                             │
+     * </pre>
+     * <p>
+     */
```
Comment (mxsm) [inline store/src/main/java/org/apache/rocketmq/store/ha/DefaultHAConnection.java:47]: how about consolidation form
```
/**
     * Transfer Header buffer size. Schema: physic offset and body size. Format:
     *
     * <pre>
     * ┌───────────────────────────────────────────────┬───────────────────────┐
     * │                  physicOffset                 │         bodySize      │
     * │                    (8bytes)                   │         (4bytes)      │
     * ├───────────────────────────────────────────────┴───────────────────────┤
     * │                                                                       │
     * │                           Transfer Header                             │
     * </pre>
     * <p>
     */
```

File: store/src/main/java/org/apache/rocketmq/store/ha/DefaultHAClient.java [hunk 3]
```diff
@@ -35,11 +35,26 @@
...
+    /**
+     * Report header buffer size. Schema: slaveMaxOffset. Format:
+     *
+     * <pre>
+     * ┌───────────────────────┬───────────────────────┐
+     * │                  slaveMaxOffset               │
+     * │                    (8bytes)                   │
+     * ├───────────────────────┴───────────────────────┤
+     * │                                               │
+     * │                  Report Header                │
```
Comment (mxsm) [inline store/src/main/java/org/apache/rocketmq/store/ha/DefaultHAClient.java:47]: So is this place.

**Review by mxsm — CHANGES_REQUESTED** [review]
reference AutoSwitchHAConnection  class comments

File: store/src/main/java/org/apache/rocketmq/store/ha/DefaultHAConnection.java (hunk 2 above)
Comment (TheR1sing3un) [reply store/src/main/java/org/apache/rocketmq/store/ha/DefaultHAConnection.java:47]: > how about consolidation form
> 
> ```
> /**
>      * Transfer Header buffer size. Schema: physic offset and body size. Format:
>      *
>      * <pre>
>      * ┌───────────────────────────────────────────────┬───────────────────────┐
>      * │                  physicOffset                 │         bodySize      │
>      * │                    (8bytes)                   │         (4bytes)      │
>      * ├───────────────────────────────────────────────┴───────────────────────┤
>      * │                                                                       │
>      * │                           Transfer Header                             │
>      * </pre>
>      * <p>
>      */
> ```

sounds good!

**Review by Oliverwqcwrw — APPROVED** [review]

**Comment by RongtongJin** [comment]
Hi @TheR1sing3un, plz merge the latest develop branch to pass CI.

**Review by RongtongJin — APPROVED** [review]

**Pull request merged**